Documentation/ColumnsConfig/Type/Color/_Properties/_ValuePicker.rst.txt

```diff
@@ -7,7 +7,12 @@
     :Scope: Display
 
     Renders a select box with static values next to the input field. When a
-    value is selected in the box, the value is transferred to the field. Keys:
+    value is selected in the box, the value is transferred to the field.
+
+    ..  versionchanged:: 14.0
+        It is now possible to define associative array keys for the :php:`items`
```

Comment:
it is now **required**, also below

Reply:
@georgringer I created a follow-up: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/pull/1249